docs/pages/doc-status.md

```diff
@@ -0,0 +1,11 @@
+# XMTP documentation status
+
+We’re excited to share documentation for the latest version of XMTP.
```

Comment:
this is exactly the right framing for v3!

Reply:
Thank you for your patience and guidance around this @saulmc!